src/fusor/fusor.py

```diff
@@ -318,21 +313,21 @@ async def transcript_segment_element(
 
     def gene_element(
         self, gene: str, use_minimal_gene_descr: bool = True
-    ) -> tuple[GeneElement | None, str | None]:
+    ) -> tuple[Gene | None, str | None]:
         """Create gene element
 
         :param str gene: Gene
-        :param bool use_minimal_gene_descr: `True` if minimal gene descriptor
+        :param bool use_minimal_gene_descr: `True` if minimal gene object
             (`id`, `gene_id`, `label`) will be used. `False` if
-            gene-normalizer's gene descriptor will be used
+            gene-normalizer's gene object will be used
         :return: GeneElement, warning
         """
-        gene_descr, warning = self._normalized_gene_descriptor(
+        normalized_gene, warning = self._normalized_gene(
```

Comment:
I believe we can simplify and just return here (removing lines after) since it does the same
```suggestion
        return self._normalized_gene(
```